sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java

```diff
@@ -28,15 +31,8 @@ synchronized ManagedChannel getChannel() {
             return channel;
         }
 
-        var channelBuilder = ManagedChannelBuilder.forTarget(address);
```

Comment:
This is my bad call. I thought we can remove the conditional on if address ends with `:50212` or `:433`, but after the weekly meeting I realized this is meat to stay in the SDK; so can you undo this change?